pages/quantum.js

```diff
@@ -60,12 +60,12 @@ import ControlledDialog from '../components/ControlledDialog';
 
 export default function Quantum() {
   return (
-    <div className="ComponentsContainer">
+    <div className="">
```

Comment:
```suggestion
    <>
```
As the sole purpose of this div is to group elements, what do you think about replacing it with a "fragment"(<></>)?